echopype/tests/test_convert.py

```diff
@@ -14,7 +14,7 @@
 import xarray as xr
 import pytest
 from pathlib import Path
-from ..convert import open_raw
+from ..echodata import open_raw
```

Comment:
For tests, it's better if you don't do relative paths since `echodata` should be installed before you run pytest. So you can just do `from echodata import ...`

Reply:
I see, cool, thanks for explaining this!